packages/arb-token-bridge-ui/src/hooks/useOftTransactionHistory.ts

```diff
@@ -302,7 +308,16 @@ export function useOftTransactionHistory({
           isTestnet ? LAYERZERO_API_URL_TESTNET : LAYERZERO_API_URL_MAINNET
         }/messages/wallet/${walletAddress}`
       : null,
-    fetcher
+    fetcher,
+    {
+      onErrorRetry: (error, key, config, revalidate, { retryCount }) => {
+        // don't retry on 404 (LayerZero API returns 404 if no transactions are found)
+        if (error.status === 404 || retryCount >= 2) return
+
+        // else, retry on error as usual
+        setTimeout(() => revalidate({ retryCount }), 10000)
```

Comment:
We could retry after the default 5 seconds.
https://swr.vercel.app/docs/error-handling.en-US#error-retry
```suggestion
        setTimeout(() => revalidate({ retryCount }), 5_000)
```